docs/api-reference/full-text/index.mdx

```diff
@@ -363,6 +363,22 @@ The following tokenizer names are accepted by the `paradedb.tokenizer` function.
   Equivalent to `stem` with language set to `English`.
 </ParamField>
 
+#### Pick a Tokenizer
+
+If you're not sure what tokenizer to use, try `paradedb.tokenize` with different tokenizers.
```

Comment:
If you're not sure which tokenizer to use, try `paradedb.tokenize` with a different tokenizers to see how the input text gets tokenized.

Reply:
or:

`paradedb.tokenize` allows the user to evaluate different tokenization strategies.

Reply:
I like that!